packages/contractkit/src/governance/proposals.ts

```diff
@@ -5,21 +5,22 @@ import { Transaction, TransactionObject } from 'web3/eth/types'
 import { CeloContract } from '../base'
 import { obtainKitContractDetails } from '../explorer/base'
 import { BlockExplorer } from '../explorer/block-explorer'
-import { ABI as GovernanceABI } from '../generated/Governance'
 import { ContractKit } from '../kit'
 import { CeloTransactionObject, valueToString } from '../wrappers/BaseWrapper'
 import { GovernanceWrapper, Proposal, ProposalTransaction } from '../wrappers/Governance'
 import { setImplementationOnProxy } from './proxy'
 
-export const PROPOSE_PARAM_ABI_TYPES = (GovernanceABI.find(
-  (abiEntry) => abiEntry.name! === 'propose'
-)!.inputs! as Array<{ type: string }>).map((abiInput) => abiInput.type)
+export const PROPOSE_PARAM_ABI_TYPES = ['uint256[]', 'address[]', 'bytes', 'uint256[]']
```

Comment:
i think it's cleaner to use the `GovernanceABI.find()` strategy, since it's more resilient to future changes. In fact you can extract a function `getAbiTypes(ABI, methodName)`